tee-worker/omni-executor/parentchain/api-interface/README.md

```diff
@@ -0,0 +1,43 @@
+# Parentchain API Interface
+
+This crate contains auto-generated Substrate runtime API definitions for the OmniAccount and Teebag pallets.
+
+## Overview
+
+The crate exports the generated interface from a scale encoded metadata file. The interface is used to interact with the parentchain's runtime API.
+
+## Generation Process
```

Comment:
Could you write a bit **when** it should be re-generated/updated?

Also does it imply a potential inconsistency if e.g. the interface is generated against local chain but live chain has an older interface so worker won't work along?

Reply:
Correct, we have to keep the version in mind. But this should not be an issue, we can re-generate the interface as part of our build process using the target chain endpoint on CI, it won't compile if the types are not compatible. 

I'll add some comments in the docs